src/views/TabView/TabBarBottom.js

```diff
@@ -71,10 +71,18 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
       inputRange,
       outputRange,
     });
+    const label = this.props.getLabelText(scene);
+    if (typeof label === 'string') {
+      return (
+        <Animated.Text style={[styles.label, { color }]}>
+          {label}
+        </Animated.Text>
+      );
+    }
     return (
```

Comment:
@satya164 Do you mean like:

```
if (typeof label === 'string') {
  return label;
``` 

Won't this be missing the default styles though?